odoo/fields.py

```diff
@@ -1607,6 +1607,11 @@ def get_trans_terms(self, value):
         self.translate(terms.append, value)
         return terms
 
+    def get_text_content(self, term):
+        """ Return the textual content for the given term. """
+        func = getattr(self.translate, 'get_text_content', None)
```

Comment:
```suggestion
        func = getattr(self.translate, 'get_text_content', lambda term: term)
```
would avoid the need for the conditional on the next line.